test/fixtures/es-module-loaders/builtin-named-exports-loader.mjs

```diff
@@ -28,7 +28,7 @@ export function load(url, context, next) {
       source: generateBuiltinModule(urlObj.pathname),
       format: 'commonjs',
     };
-  } else if (context.format === 'commonjs') {
+  } else if (context.format === undefined || context.format === 'commonjs') {
```

Comment:
```suggestion
  } else if (context.format == null || context.format === 'commonjs') {
```

Reply:
You marked this as resolved, but I’d like to see it committed. We don’t want our users to use strict equality (because both `null` and `undefined` should be treated the same), so we shouldn’t write our tests with strict equality

Reply:
Wouldn't this make our tests less specific, and therefore less able to alert us to changes that might affect our users?

Reply:
I think that's fine, if we want more specific tests, we have the ones that checks the output. For the loaders in fixtures, we should write them in a way we want our users to get inspiration from.